docs/taskruns.md

```diff
@@ -7,33 +7,39 @@ weight: 300
 
 # `TaskRuns`
 
-- [Overview](#taskruns)
+<!-- toc -->
+- [Overview](#overview)
 - [Configuring a `TaskRun`](#configuring-a-taskrun)
   - [Specifying the target `Task`](#specifying-the-target-task)
   - [Tekton Bundles](#tekton-bundles)
   - [Specifying `Parameters`](#specifying-parameters)
+    - [Implicit Parameters](#implicit-parameters)
+    - [Extra Parameters](#extra-parameters)
   - [Specifying `Resources`](#specifying-resources)
-  - [Specifying `ServiceAccount` credentials](#specifying-serviceaccount-credentials)
+  - [Specifying `Resource` limits](#specifying-resource-limits)
   - [Specifying a `Pod` template](#specifying-a-pod-template)
   - [Specifying `Workspaces`](#specifying-workspaces)
   - [Specifying `Sidecars`](#specifying-sidecars)
   - [Specifying `LimitRange` values](#specifying-limitrange-values)
-  - [Configuring the failure timeout](#configuring-the-failure-timeout)
+- [Configuring the failure timeout](#configuring-the-failure-timeout)
```

Comment:
thanks @nbaudis for the updates 🙏 

This can be placed under Configuring a `TaskRun`. Please update the section header so that it aligns with the other "Specifying*" titles.